datasets/fashion_mnist/pipelines/fashion_mnist/fashion_mnist_dag.py

```diff
@@ -0,0 +1,132 @@
+        env_vars={
+            "SOURCE_URL": "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-images-idx3-ubyte.gz",
+            "SOURCE_FILE": "files/train-images-idx3-ubyte.gz",
+            "TARGET_FILE": "files/train-images-idx3-ubyte.gz",
+            "TARGET_GCS_BUCKET": "{{ var.value.composer_bucket }}",
+            "TARGET_GCS_PATH": "data/fashion_mnist/fashion_mnist_train/train-images-idx3-ubyte.gz",
+            "PIPELINE_NAME": "fashion_mnist",
+        },
+        resources={
+            "request_memory": "2G",
+            "request_cpu": "200m",
+            "request_ephemeral_storage": "8G",
+        },
+    )
+
+    # Task to copy `train-labels-idx1-ubyte.gz` from FASHION MNIST  Database to GCS
+    download_and_process_source_zip_file_3 = kubernetes_pod.KubernetesPodOperator(
+        task_id="download_and_process_source_zip_file_3",
+        name="fashion_mnist",
+        namespace="composer",
+        service_account_name="datasets",
+        image_pull_policy="Always",
+        image="{{ var.json.fashion_mnist.container_registry.run_csv_transform_kub }}",
+        env_vars={
+            "SOURCE_URL": "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-labels-idx1-ubyte.gz",
+            "SOURCE_FILE": "files/train-labels-idx1-ubyte.gz",
+            "TARGET_FILE": "files/train-labels-idx1-ubyte.gz",
+            "TARGET_GCS_BUCKET": "{{ var.value.composer_bucket }}",
+            "TARGET_GCS_PATH": "data/fashion_mnist/fashion_mnist_train/train-labels-idx1-ubyte.gz",
+            "PIPELINE_NAME": "fashion_mnist",
+        },
+        resources={
+            "request_memory": "2G",
+            "request_cpu": "200m",
+            "request_ephemeral_storage": "8G",
+        },
+    )
+
+    # Task to copy `t10k-labels-idx1-ubyte.gz` from FASHION MNIST  Database to GCS
+    download_and_process_source_zip_file_4 = kubernetes_pod.KubernetesPodOperator(
+        task_id="download_and_process_source_zip_file_4",
+        name="fashion_mnist",
+        namespace="composer",
+        service_account_name="datasets",
+        image_pull_policy="Always",
+        image="{{ var.json.fashion_mnist.container_registry.run_csv_transform_kub }}",
+        env_vars={
+            "SOURCE_URL": "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-labels-idx1-ubyte.gz",
+            "SOURCE_FILE": "files/t10k-labels-idx1-ubyte.gz",
+            "TARGET_FILE": "files/t10k-labels-idx1-ubyte.gz",
+            "TARGET_GCS_BUCKET": "{{ var.value.composer_bucket }}",
+            "TARGET_GCS_PATH": "data/fashion_mnist/fashion_mnist_test/t10k-labels-idx1-ubyte.gz",
+            "PIPELINE_NAME": "fashion_mnist",
+        },
+        resources={
+            "request_memory": "2G",
+            "request_cpu": "200m",
+            "request_ephemeral_storage": "8G",
+        },
+    )
+
+    (
+        download_and_process_source_zip_file
```

Comment:
Can we use a different naming convention for these nodes, other than using _2, _3, _4? Maybe a descriptive name of what each node does?